.dockstore.yml

```diff
@@ -0,0 +1,29 @@
+version: 1.2
+tools:
+  - subclass: CWL
+    primaryDescriptorPath: /bamstats.cwl
+    testParameterFiles:
+      - /test.json
+    name: bamstats_cwl
+    authors:
+      - orcid: 0000-0002-6130-1021
+  - subclass: CWL
+    primaryDescriptorPath: /bamstats_sort.cwl
+    testParameterFiles:
+      - /test.json
+    name: bamstats_sort_cwl
+    authors:
+      - orcid: 0000-0002-6130-1021
+workflows:
+  - subclass: WDL
+    primaryDescriptorPath: /bamstats.wdl
+    testParameterFiles:
+      - /test.wdl.json
+    name: wdl
```

Comment:
workflow name should be "bamstats_wdl" or something like that

Reply:
Seems redundant. The name of the tool is already `github.com/dockstore/dockstore-tool-bamstats/wdl`

Reply:
I think most people view the last part as the de facto name of the workflow, even though strictly speaking that's not true, and if it's redundancy we're worried about I think "dockstore-tool" is the bigger issue.

Reply:
There's also the inconsistency -- we put bamstats in the names of the CWLs. Why not do the same with the WDL and NFL?

(Note that this is a nitpick at the moment and I'm not willing to die on this hill, but depending if/how we address https://github.com/dockstore/dockstore/issues/4790 and https://github.com/dockstore/dockstore/issues/4491 this could have implications later.)